ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -0,0 +1,307 @@
+
+  ![Disable workflow](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/DisableWorkflow.png "[Disable workflow]")
+
+5. Click the flag at the bottom of the engine room.    
+  The engine room panel is folded.
+
+6. Edit the number of workers in the *Number of workers* field.
+
+7. Click the [SAVE] button in the upper right corner.    
+  The changes are saved. The *Edit queue type* view is closed. The *QueueType has been saved* pop-up window is displayed.  
+
+  ![Queue type saved](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/QueueTypeSaved.png "[Queue type saved]")
+
+8. Unfold the engine room panel, disable the *Disable Workflow* toggle in the *Actindo Work Flow Engine* box and fold the engine room panel again.
+
+  ![Enable workflow](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/EnableWorkflow.png "[Enable workflow]")
+
+[comment]: <> (Soll das überhaupt beschrieben werden? Wegen entwicklermodus und so...)
+
+### Next steps
+
+- [Assign a queue type](#assign-a-queue-type)
+- [Delete a queue type](#delete-a-queue-type)
+- [Kill workers](#kill-workers)
+
+### See also
+
+- [User Interface workflows](/ActindoWorkFlow/UserInterface/00_UserInterface.md)
+- [Create a queue type](#create-a-queue-type)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+- [Manage the workflows](/ActindoWorkFlow/Operation/01_ManageWorkflows.md)
+
+
+## Assign a queue type
+
+Assign a certain queue type to an action, for instance to prioritize a certain action to be executed faster.
+
+### Prerequisites
+
+At least one queue type is created, see [Create a queue type](#create-a-queue-type).
+
+### Procedure
+
+*Workflows > Workflows > Tab OVERVIEW > Select workflow > Select workflow version*
+
+![Workflow editor](/Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditor.png "[Workflow editor]")
+
+1. Select the action you want to assign a certain queue type to.      
+The action is highlighted and its settings are displayed in the settings side bar on the right.
+
+  ![Action](/Assets/Screenshots/ActindoWorkFlow/Workflows/Action.png "[Action]")
+
+2. Click the *Queue Type* drop-down list and select the desired queue type for the selected action. All available queue types are displayed in the list.     
+The selected queue type is displayed in the list.
+
+> [Info] The changes are automatically saved.
+
+[comment]:  <> (Is that right? Or do I have to click the deploy button to apply these changes?)
+
+### Next steps
+
+- [Delete a queue type](#delete-a-queue-type)
+- [Kill workers](#kill-workers)
+
+### See also
+
+- [User Interface workflows](/ActindoWorkFlow/UserInterface/00_UserInterface.md)
+- [Create a queue type](#create-a-queue-type)
+- [Edit a queue type](#edit-a-queue-type)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+- [Manage the workflows](/ActindoWorkFlow/Operation/01_ManageWorkflows.md)
+
+
+
+## Delete a queue type
```

Comment:
Have to check prerequisits. I would assume the queue type must not be assigned to any action anymore, additionally i think the workers have to be stopped again